conf/server.properties

```diff
@@ -92,7 +92,7 @@ org.dbpedia.spotlight.candidateMap.loadToMemory = true
 #----- DISAMBIGUATION -------
 
 # List of disambiguators to load: Document,Occurrences,CuttingEdge,Default
-org.dbpedia.spotlight.disambiguate.disambiguators = CuttingEdge
+org.dbpedia.spotlight.disambiguate.disambiguators = Default,Document
```

Comment:
I don't know how/why that happened. I don't really use server.properties anyway. reverting this